src/symmetricDifference.js

```diff
@@ -0,0 +1,25 @@
+var _curry2 = require('./internal/_curry2');
+var difference = require('./difference');
+var union = require('./union');
+
+
+/**
+ * Finds the set (i.e. no duplicates) of all elements contained in the first or
+ * second list, but not both.
+ *
+ * @func
+ * @memberOf R
+ * @category Relation
+ * @sig [a] -> [a] -> [a]
+ * @param {Array} list1 The first list.
+ * @param {Array} list2 The second list.
+ * @return {Array} The elements in `list1` or `list2`, but not both.
+ * @see R.symmetricDifferenceWith
+ * @example
+ *
+ *      R.symmetricDifference([1,2,3,4], [7,6,5,4,3]); //=> [1,2,7,6,5]
+ *      R.symmetricDifference([7,6,5,4,3], [1,2,3,4]); //=> [7,6,5,1,2]
+ */
+module.exports = _curry2(function symmetricDifference(list1, list2) {
+  return union(difference(list1, list2), difference(list2, list1));
```

Comment:
Obviously we should use `concat` for performance reasons, but it seems so much less elegant that way.  :frowning: